images/make-dind/Dockerfile

```diff
@@ -73,7 +73,8 @@ RUN apt-get update \
     && update-alternatives --set ip6tables /usr/sbin/ip6tables-legacy
 
 # Move Docker's storage location & enable experimental features & add Container Registry cache (see: https://cloud.google.com/container-registry/docs/pulling-cached-images)
-RUN echo 'DOCKER_OPTS="${DOCKER_OPTS} --data-root=/docker-graph --experimental --registry-mirror=https://mirror.gcr.io"' | \
+# @inteon: added --mtu 1400 to fix network issues due to parent mtu < child mtu (see https://blog.zespre.com/dind-mtu-size-matters.html)
+RUN echo 'DOCKER_OPTS="${DOCKER_OPTS} --mtu=1400 --data-root=/docker-graph --experimental --registry-mirror=https://mirror.gcr.io"' | \
```

Comment:
suggestion: Is it worth adding in this comment what the parent MTU was at the time the flag was added, which would help explain to people reading the code why the value of the MTU here was chosen? This comment explains why we set the MTU but doesn't explain the value we chose.

question: Why 1400? If the parent is 1460 we're potentially wasting a bunch of packets by setting the MTU to 1400 here. 1460 should be safe while being more efficient, no?

Reply:
I randomly picked 1400 so the fix would still work if the parent mtu changed to eg. 1450.
I'll already add a note stating that the mtu was 1460 at the time of writing this fix.

Reply:
UPDATE: I found that the 1460 is a well-defined value: https://cloud.google.com/kubernetes-engine/docs/concepts/network-overview

I updated the mtu value to 1460 (matching the parent mtu).
I also added a comment to explain that this was the value at the time of writing.